requirements.txt

```diff
@@ -44,7 +44,6 @@ google-cloud-bigquery==1.26.0
 
 pinax-eventlog==2.0.3
 ptvsd==4.3.2
-django-ptvsd-debug==1.0.3
```

Comment:
since you removed it we need to remove from installed app as well otherwise the build will fail

```
INSTALLED_APPS = (
    'django_ptvsd',
    ...
)
```

Reply:
Why is this package is not needed?

Reply:
i am guessing since we are using using Gunicorn with one worker and not the django default server?

Reply:
This is no longer needed because most of the code that was in this I put into ptvsd_util.py. This package added a django-admin management command which didn't work with gunicorn, only with the runserver.

Reply:
Thanks for noticing this, I've removed it from INSTALLED_APPS.